app/views/users/show.html.erb

```diff
@@ -1 +1,47 @@
 <h2>This is the details of the specific user you selected</h2>
+
+<div>
+   <div class='user_info'>
+    <img src="<%= @user.photo%>" alt="<%= @user.name%>" class='user_image'>
+    <div class='user_details'>
+      <h2>
+        <%= @user.name%>
+      </h2>
+      <h4>
+        <%= @user.posts_counter%>
+      </h4>
+    </div>
+  </div>
+    <div class='user_bio'>
+      <%= @user.bio %>
+    </div>
+
+    <div class='posts'>
+      <% @user.recent_posts.each do |post|%>
+      <div class='post'>
+        <h4 class='post_title'>
+          <%= post.title %>
+        </h4>
+        <p class='post_body'>
+          <%= post.body %>
+        </p>
+        <span class='comments_likes'>
+           <p>
+            Comments:
+            <%= post.comments_counter %>
+          </p>
+           <p>
+            Likes:
+            <%= post.likes_counter %>
+          </p>
+        </span>
+      </div>
+      <% end %>
+      <div class='btn_container'>
+        <%= link_to [@user, :posts] do %>
+          <button class='all_posts_btn'>See all Posts</button>
+        <%end%>
+      </div>
+    </div>
```

Comment:
- [ ] @habasm great work on your user show page, however therre are some issues that need to fixed:

   - [ ] The number of posts is placed wrongly and doesn't have a text indicating the number of posts as shown in the comparison below
   - [ ] The page margin is so small kindly try an increase it, this applies to all other pages.
   - [ ] The biography section doesn't have a closing on the right, it's suppose to be enclosed in a rectangle like posts elements

Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/habasm/BlogApp/assets/61408860/536b4c57-eb53-468d-8f24-6ff392f6ff65)  |![image](https://github.com/habasm/BlogApp/assets/61408860/0f356158-ccf9-460d-a0eb-a8645172629f)